phaseshifts/leed.py

```
@@ -192,22 +180,15 @@ def validate(self, filename, aoi=False):
                 sys.stderr.flush()
         # unit cell
         try:
-            basis = [
-                line
-                for line in lines
-                if line.startswith("a") and int(line[1]) <= 3 and int(line[1]) > 0
-            ]
-
+            _basis = [
+                line for line in lines if line.startswith("a") and int(line[1]) <= 3 and int(line[1]) > 0
+            ]  # noqa: F841
```

Ruff (0.14.10)

Unused `noqa` directive (unused: `F841`)

Remove unused `noqa` directive

(RUF100)